csharp-selenium-webdriver-sample/WebDriverFactory.cs

```diff
@@ -21,13 +22,15 @@ public static IWebDriver GetWebDriver(BrowserType browser)
             switch (browser)
             {
                 case BrowserType.Chrome:
-                    // Check if Chrome web driver environment variable already defined so that Chrome version and the webdriver are the same, otherwise use Selenium.WebDriver.ChromeDriver to install it                    
+                    //  Check if Chrome web driver environment variable already defined so that Chrome version and the webdriver are the same, otherwise use Selenium.WebDriver.ChromeDriver to install it                    
```

Comment:
This comment is one of the better ones in that it's describing some of *why* the line of code exists, rather than just repeating what it's doing. I think it would be nice to expand this a little (how exactly does the environment variable ensure that the chrome version is in sync? what sets the environment variable? in what way does saying "CurrentDirectory" cause Selenium.WebDriver.ChromeDriver to install it?). You should be able to find a very similar line of code in the typescript sample that would be a good starting point